src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -4,31 +4,91 @@
 
 package frc.robot.subsystems;
 
+
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
 import frc.robot.status.Status;
 import frc.robot.status.Status.IntakeStatus;
-import frc.robot.util.SparkMax;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.I2C.Port;
+import com.revrobotics.ColorSensorV3;
+
 
 public class IndexerSubsystem extends SubsystemBase {
   /** Creates a new IndexerSubsystem. */
-  public IndexerSubsystem() {}
-
   CANSparkMax belt = new CANSparkMax(9, MotorType.kBrushless);
   CANSparkMax transition = new CANSparkMax(8, MotorType.kBrushless);
 
+  private static ColorSensorV3 sensor;
+  private static I2C.Port I2C;
 
-  public void spinMotors(double speed) {
+  public IndexerSubsystem() {
+    sensor = new ColorSensorV3(Port.kOnboard);
+  }
+
+  public void spinBelt(double speed) {
     belt.set(-speed);
```

Comment:
invert motor controller instead of setting to a negative value - positive should always be the direction that you want it to spin